.changeset/bright-gifts-melt.md

```diff
@@ -0,0 +1,5 @@
+---
+'@chainlink/ea-bootstrap': patch
+---
+
+Delete normalizeStatusCode function
```

Comment:
Let's clarify this a bit

```suggestion
'@chainlink/ea-bootstrap': minor
---

Status codes are no longer normalized (e.g. 429s have the metric label '429' instead of '4xx')
```